cf-agent/nfs.c

```diff
@@ -126,13 +127,36 @@ static const char *const VMOUNTOPTS[] =
     [PLATFORM_CONTEXT_ANDROID] = "defaults",         /* android */
 };
 
+
+void GetHostAndSource(char *buf, char *host, char *source)
+/* Extracts from buffer host & source of cifs and panfs network systems */
+{
+    int i = 0, i2 = 0, bs = 0;
```

Comment:
So `bs` is used to count slashes, and `i2` is used as an index for `source`. I would call these variables `slashes` and `source_index`, respectively.